src/main/java/com/bootcamp/demo/model/BookScooter.java

```diff
@@ -0,0 +1,59 @@
+package com.bootcamp.demo.model;
+
+public class BookScooter {
+    private String id;
```

Comment:
Could you outline what this id represents?

Reply:
in fact, I'm going to delete it because we don't need it